src/cljs/velho_ds/organisms/grid.cljs

```diff
@@ -0,0 +1,24 @@
+(ns velho-ds.organisms.grid
+  (:require [stylefy.core :as stylefy]))
+
+(defn- style-grid [{:keys [rows cols]}]
```

Comment:
Tyylit sovittiin laitettavan erilliseen tiedostoon. Nyt en o ihan varma, että onko siitä tässä tapauksessa hyötyä. Mitä mieltä, jos tämän (grid) tapauksessa tää ainoo tyyli menis näin?